### ISSUE TITLE
Omit specific fields like hostname

### ISSUE THREAD

**Issue by wachunga** [trentm/node-bunyan#52]
My log output is overly verbose and I would like to reduce the included metadata, but it's not clear to me how to omit hostname, pid, etc. Could included fields be made configurable? 

> [2012-10-29T21:55:53.087Z]  INFO: myLogger/86281 on hostname.local (/Users/me/dev/src/project/server/dao.js:235 in bigMethodName): something happened

If fields are configurable, could that be documented more clearly?

**Comment by gpsinghsandhu** [comment]
I know it has been a long time but I ended up here because people at our company wanted to see nicer message in bunyan logs and suppress some fields. So taking inspiration from @mpareja 's comment [here](https://github.com/trentm/node-bunyan/issues/271), I created this small package - [tunyan](https://www.npmjs.com/package/tunyan).

This is a first draft and I will keep updating it, hope it helps someone.
